rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -50,7 +49,7 @@
 class ConveRTFeaturizer(DenseFeaturizer, GraphComponent):
     """Featurizer using ConveRT model.
 
-    Loads the ConveRT(https://github.com/PolyAI-LDN/polyai-models#convert)
+    Loads the ConveRT("https://github.com/davidalami/ConveRT/releases/download/1.0/nocontext_tf_model.tar.gz")
```

Comment:
this should not be a model download location but a place where to find information about the model type. It should be updated, but probably to this link: https://github.com/davidalami/ConveRT